api/util/drf_writable_nested/serializers.py

```diff
@@ -0,0 +1,34 @@
+from drf_writable_nested import NestedCreateMixin, NestedUpdateMixin
+from rest_framework import serializers
+
+
+class NestedUpdateMixinDeleteBeforeUpdate(NestedUpdateMixin):
```

Comment:
I guess maybe we should wait for the outcome of your discussion on the github issue you linked below before merging this?